src/types/markers.js

```diff
@@ -319,6 +333,15 @@ export type NetworkPayload = {|
   responseEnd?: Milliseconds,
 |};
 
+export type DiskIoPayload = {|
```

Comment:
The part to add this type should be moved to the PR #1687

Reply:
The test includes the `DiskIoPayload,` and is needed for proper type checking.

```js
     expect(contentMarkers[12]).toEqual({
      data: {
        // Stack property is converted to a cause.
        cause: {
          stack: 2,
          time: 1,
        },
        endTime: 1024,
        filename: '/foo/bar/',
        source: 'PoisionOIInterposer',
        startTime: 1022,
        type: 'io',
      },
      dur: 2,
      name: 'create/open',
      start: 1022,
      title: null,
    });
```